package.json

```
@@ -1,108 +1,9 @@
 {
-  "name": "@react-native-async-storage/async-storage",
-  "version": "1.19.2",
-  "description": "Asynchronous, persistent, key-value storage system for React Native.",
-  "main": "lib/commonjs/index.js",
-  "module": "lib/module/index.js",
-  "react-native": "src/index.ts",
-  "types": "lib/typescript/index.d.ts",
-  "files": [
-    "RNCAsyncStorage.podspec",
-    "android/build.gradle",
-    "android/src",
-    "android/testresults.gradle",
-    "ios/",
-    "jest/",
-    "lib/",
-    "macos/",
-    "src/",
-    "windows/"
+  "name": "@react-native-async-storage/project",
```

Some tools (like Nx) use the convention `@scope/root`:
```suggestion
  "name": "@react-native-async-storage/root",
```

---

sounds good 👍